apps/dotcom/src/utils/sharing.ts

```diff
@@ -70,7 +70,9 @@ async function getSnapshotLink(
 		return ''
 	}
 	const paramsToUse = getViewportUrlQuery(editor)
-	const params = paramsToUse ? `?${new URLSearchParams(paramsToUse).toString()}` : ''
+	const params = paramsToUse
```

Comment:
btw, we should add a comment (here and below) - this works but it's all a bit cursed. i think we're accommodating react-router to the fact that it doesn't encode commas, specifically.

_technically_, the commas _should_ be encoded in the URL so, because we're doing the decode here we're trying to undo what URLSearchParams is meant to do correctly, ugh. 🤦

Reply:
Yeah. I did a quick search and it seems react router has a bunch of encoding issues.